### PR TITLE
Simplify HelloWorld tutorial including generic client invocation

### PR DESCRIPTION
**Pull request by EugeneYYY**
- Modified plug-in.cpp/h to minimize number of required changes and make code more concise
- Modified generic client to 
    - allow multiple input string 
    - decrease amount of output (by default decrypted JSON output and receipts are not shown)
    - set a default URI value 
- Updated related README.md files

Signed-off-by: EugeneYYY <yevgeniy.y.yarmosh@intel.com>

### CODE REVIEW

File: docs/workload-tutorial/hello_world/stage_2/plug-in.h [hunk 1]
```diff
@@ -19,9 +19,14 @@
...
 
-class HelloWorld : public WorkloadProcessor {
+namespace HelloWorld {
+
+const char* workload_id="hello-world";
+
+class Workload : public WorkloadProcessor {
 public:
-    IMPL_WORKLOAD_PROCESSOR_CLONE(HelloWorld)
+    // Replace $CLASS_NAME$ with an appropriate class name, e.g. HelloWorld
```
Comment (manojgop) [inline docs/workload-tutorial/hello_world/stage_2/plug-in.h]: There is no $CLASS_NAME$ variable here. So it can  be confusing

**Comment by manojgop** [comment]
Could you rebase and push updated commit

File: examples/apps/generic_client/generic_client.py [hunk 2]
```diff
@@ -47,14 +47,17 @@ def ParseCommandLine(args) :
 	global mode
 	global uri
 	global address
+	global show_receipt
+	global show_decrypted_output
```
Comment (manojgop) [inline examples/apps/generic_client/generic_client.py]: Number of global variables ins increasing :-). We could probably raise another PR to reorganize things a bit.

File: docs/workload-tutorial/hello_world/stage_2/plug-in.h (hunk 1 above)
Comment (danintel) [reply docs/workload-tutorial/hello_world/stage_2/plug-in.h]: Fixed in 3 files:
./docs/workload-tutorial/templates/plug-in.h
./docs/workload-tutorial/hello_world/stage_2/plug-in.h
./docs/workload-tutorial/hello_world/stage_1/plug-in.h

**Comment by danintel** [comment]
In order to save time, as Eugene is busy today, I opened new PR #113  for this and I am closing this.

**Pull request closed without merging**